app/javascript/packages/document-capture/context/in-person.ts

```diff
@@ -14,6 +24,8 @@ export interface InPersonContextProps {
 
 const InPersonContext = createContext<InPersonContextProps>({
   arcgisSearchEnabled: false,
+  inPersonCtaVariantTestingEnabled: false,
+  inPersonCtaVariantActive: '',
```

Comment:
just checking my understanding - the default variant is an empty string? or does an empty string represents no variant shown?

Reply:
As defined in this file, your'e technically correct. In practice, as the app and session are initialized these values are overwritten by the application config values and AB test bucketing logic. The default bucket for this AB test is `in_person_variant_a` which is described in `ab_tests.rb`.